dispatch-rules/advies-budgetwijziging.js

```diff
@@ -32,11 +32,11 @@ let rule = {
         BIND(${sparqlEscapeUri(sender)} as ?sender)
         BIND(${sparqlEscapeUri(submission)} as ?submission)
 
-        ?submission a meb:Submission;
+        ?submission
```

Comment:
Yes, it's weird. By removing (redundant) type checking, some queries are much faster all of a sudden. It feels counter intuitive, because type checking is such a basic thing that I assumed there would have been optimisations in place for that in Virtuoso.